libraries/abstractions/ble_hal/test/src/iot_test_ble_hal_afqp.c

```diff
@@ -640,10 +653,17 @@ TEST( Full_BLE, BLE_Property_WriteLongCharacteristic )
             IotTestBleHal_WriteResponse( bletestATTR_SRVCB_CHAR_A, xWriteEvent, true );
         }
 
-        IotTestBleHal_WriteCheckAndResponse( bletestATTR_SRVCB_CHAR_A,
-                                             true,
-                                             true,
-                                             bletestsMTU_SIZE1 - 5 );
+        usOffset += _bletestsMTU_SIZE - 5;
```

Comment:
Why "- 5"? That looks like it should be a #define.

Reply:
Header is 5 bytes. I can add a macro for it.